docs/source/usersguide/input.rst

```diff
@@ -170,6 +172,34 @@ should be performed. It has the following attributes/sub-elements:
 
     *Default*: None
 
+  :keff_trigger:
+    This tag specifies a precision trigger on the combined :math:`k_{eff}`. The
+    trigger is a convergence criterion on the uncertainty of the estimated
+    eigenvalue. It has the following attributes/sub-elements:
+
+    :type:
+      The type of precision trigger. Accepted options are "variance", "std_dev",
+      and "rel_err".
+
+      :variance:
+       Variance of the batch mean :math:`\sigma^2`
+
+      :std_dev:
+        Standard deviation of the batch mean :math:`\sigma`
+
+      :rel_err:
+        Relative error of the batch mean :math:`\frac{\sigma}{\mu}`
+
+        *Default*: None
```

Comment:
Actually just realized that the indentation was correct at four rather than two - the ``threshold`` is a subelement of ``keff_trigger``.